partner_contact_in_several_companies/tests/test_partner_contact_in_several_companies.py

```diff
@@ -143,36 +144,35 @@ def test_05_contact_fields_sync(self):
         # Test DOWNSTREAM sync
         self.bob_contact.write({"name": "Rob Egnops"})
         self.assertEqual(
-            self.bob_job1.name, "Rob Egnops",
+            self.bob_job1.name,
+            "Rob Egnops",
         )
 
         # Test UPSTREAM sync
         self.bob_job1.write({"name": "Bob Egnops"})
         self.assertEqual(
-            self.bob_contact.name, "Bob Egnops",
+            self.bob_contact.name,
+            "Bob Egnops",
         )
 
     def test_06_ir_action(self):
-        """Check ir_action context is auto updated.
-        """
+        """Check ir_action context is auto updated."""
 
         new_context_val = (
             "'search_show_all_positions': " "{'is_set': True, 'set_value': False}"
         )
 
-        details = self.env["ir.actions.act_window"].for_xml_id(
-            "base", "action_partner_form"
-        )
+        details = self.env.ref("{}.{}".format("base", "action_partner_form")).read()[0]
```

Comment:
Much clearer here also to just specify self.env.ref("base.action_partner_form"). Like you do a few lines below with "partner_contact_in_several_companies.action_partner_form".